docs/api-reference/layer.md

```diff
@@ -118,7 +118,40 @@ Requires `pickable` to be true.
 
 This callback will be called when the mouse clicks over an object of this deck.gl layer with a single parameter [`info`](/docs/get-started/interactivity.md#the-picking-info-object).
 
-If this callback returns a truthy value, the `hover` event is marked as handled and will not bubble up to the [`onLayerClick`](/docs/api-reference/react/deckgl.md#-onlayerclick-function-optional-) callback of the `DeckGL` canvas.
+If this callback returns a truthy value, the `click` event is marked as handled and will not bubble up to the [`onLayerClick`](/docs/api-reference/react/deckgl.md#-onlayerclick-function-optional-) callback of the `DeckGL` canvas.
+
+Requires `pickable` to be true.
+
+##### `onDragStart` (Function, optional)
```

Comment:
Can these new events be handled by a `CompositeLayer` now or still only usable from whatever is instantiating those layers?

Reply:
The composite layer will have to overwrite the `on*` props in its constructor.